components/site/docsHomePageContent.tsx

```diff
@@ -0,0 +1,374 @@
+import React from 'react';
+import { Text, Stack, Box } from '@sparkpost/matchbox';
+import { Feedback, HelpOutline, QuestionAnswer } from '@sparkpost/matchbox-icons';
+import Link from 'next/link';
+import styled from 'styled-components';
+import css from '@styled-system/css';
+import { Elixir, PHP, Java, NodeMailer, Python, Node, Go } from './icons';
+
+/*
```

Comment:
🤣

Reply:
Yeah, I like breaking things up this way because I use a viewer on the right side of my editor and with this sized ascii font I can kind of use it like a table of contents to get around the document. :-)